applications/rapid-analysis/values-base.yaml

```diff
@@ -1,6 +1,6 @@
 image:
   repository: ghcr.io/lsst-sitcom/rubintv_production
-  tag: tickets-DM-52681
+  tag: tickets-DM-52549
```

Comment:
I usually reset these to a tag after merging and tagging the package, before merging to phalanx.